### DISCUSSION TITLE
Rework the flashcard seperator/detecting system

### DISCUSSION THREAD

**Discussion by 4Source** [st3v3nmw/obsidian-spaced-repetition#1018] [General]
I think the current system for detecting flashcards has some major issues. Especially the multi line flashcards. It is not really flexible and limits in the formatting of notes.

![image](https://github.com/user-attachments/assets/dfc369f0-212c-44f7-baf0-41ad71428653)

Related issues: #1017, #1005, #1014, #942, #929, #860, #883, #831, #755, #747, #739, #732, #729, #726, #725, #714, #696, #660, #632, #629, #606, #604, #601, #600, #581, #556, #549, #528, #507, #486, #482, #471, #451, #450, #442, #426, #407, #395, #392, #387, #380, #314, #284, #254, #192, #187, #122, #103, #93, #84

I think the mass of open feature request speak for them self...

Needs to be summarized what all of this feature request are asking for. Some of them are duplicates or ask for very similar things. And then I think we can find a better system that fits for most of this.

**Comment by ronzulu** [comment]
Great that you've compiled this list. There are also these related PRs:

https://github.com/st3v3nmw/obsidian-spaced-repetition/pull/1012
https://github.com/st3v3nmw/obsidian-spaced-repetition/pull/885
https://github.com/st3v3nmw/obsidian-spaced-repetition/pull/797

**Comment by 4Source** [comment]
Thanks, I have looked over all but I have not read all descriptions. 
I would like to divide them into groups so that similar ones are together and then summarize what was requested here

**Comment by ronzulu** [comment]
@4Source I spotted the GitHub "project" functionality, and thought it might help with this.

Unfortunately, I don't have permission on this repository to create one, so I did on my repository instead.
https://github.com/users/ronzulu/projects/1

I've added all the issues and PRs that you listed above, except discussions can't be added, so I've separated them out here:
https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/883
https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/1005
https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/831
https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/725
https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/717
https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/696
https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/192

https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/406 (is this relevant?)
https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/395 (covered by https://github.com/st3v3nmw/obsidian-spaced-repetition/pull/1012 etc)

Hope this helps us be organized about this group of issues.

Let me know if you have any permission issues accessing the project on my repository.

**Reply by 4Source** [reply]
> #406 (is this relevant?)

No not really

**Reply by 4Source** [reply]
I first have to checkout projects what is possible and so on, never worked with them but great idea

**Reply by 4Source** [reply]
> I've added all the issues and PRs that you listed above, except discussions can't be added, so I've separated them out here:
> #883
> #1005
> #831
> #725
> #717
> #696
> #192

Maybe you could use the function ``create issue from discussion`` I mean there are feature requests.
![image](https://github.com/user-attachments/assets/8be51a68-3bac-4326-9369-030b155079fc)

And I have seen #406 and #717 are kind of related there are both asking for Styling. So we should ignore it too. I will remove both of them in my list.

**Reply by ronzulu** [reply]
Good idea regarding `create issue from discussion`, 

| Discussion | Note | 
|  - |  - | 
| https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/883 | Created issue | 
| https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/1005 | Awaiting response to suggestion  | 
| https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/831 | Created issue  | 
| https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/725 |  Awaiting response to suggestion  | 
| https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/696 |  Is this relevant? | 
| https://github.com/st3v3nmw/obsidian-spaced-repetition/discussions/192 | Created issue  |

**Reply by 4Source** [reply]
No I think #696 does not match
